### ISSUE TITLE
Copilot-cmp joins current line with previous line unexpectedly

### ISSUE THREAD

**Issue by Bekaboo** [zbirenbaum/copilot-cmp#53]
When I select the entry from copilot in the completion menu, copilot-cmp will sometimes join the current line with the previous line.

Minimal nvim-cmp config:

```lua
local cmp = require('cmp')

cmp.setup({
  mapping = {
    ['<C-n>'] = cmp.mapping(function(fallback)
      if vim.fn.mode() == 'i' then
        if cmp.visible() then
          cmp.select_next_item()
        else
          cmp.complete()
        end
      elseif vim.fn.mode() == 'c' then
        if cmp.visible() then
          cmp.select_next_item()
        else
          fallback()
        end
      end
    end, { 'i', 'c' }),
  },
  sources = {
    { name = 'copilot' },
  },
})
```

Copilot-cmp config:

```lua
require('copilot_cmp').setup({
  formatters = {
    label = function(item) return item.text:gsub('^%s*', '') end,
    insert_text = require('copilot_cmp.format').remove_existing
  },
})
```

Copilot.lua config:

```lua
vim.defer_fn(function()
  if vim.g.loaded_coplilot then
    return
  end
  vim.g.loaded_coplilot = true
  require('copilot').setup({
    suggestion = { enabled = false },
    panel = { enabled = false },
  })
end, 100)
```

Screen recording:

https://user-images.githubusercontent.com/76579810/226435880-4cd28a8b-3bfc-4541-8685-f2d2c284e12c.mp4

**Comment by keiqu** [comment]
Yep, I frequently have same issue.

**Comment by zbirenbaum** [comment]
Please test #67 and see if it fixes your issue

**Comment by Bekaboo** [comment]
Thanks, will install and test it this weekend

**Comment by zbirenbaum** [comment]
I've reproduced this, it is not fixed by #67. This is probably the next big issue to tackle

**Comment by zbirenbaum** [comment]
This was fixed at some point and forgot to close
